Cheatsheet.md

```diff
@@ -259,19 +227,18 @@ returns the position of the first occurrence of a specified value in a string.
 Number, representing the position where the specified searchvalue occurs for the first time, or -1 if it never occurs.
 
 **Example:**
-
+```
     print('Departed Train'.indexOf('Train'));
+```
 *Output:*
     
     9
 
 ---
 ---

```

Comment:
Are these lines required ?

Reply:
These are two horizontal lines separating the header contents. Earlier the size was smaller for the headings that is why I had inerted these. Should I remove them now?

Reply:
Yeah, remove them.
And I said to remove extra line space, you remove all of them.
Please fix that too.

Reply:
Ok.